src/shared/lib/resources/opportunity/code-with-us.ts

```diff
@@ -51,10 +51,7 @@ export interface CWUOpportunity {
   createdBy?: UserSlim;
   updatedBy?: UserSlim;
 
-  // TODO
-  // @successful-proponent
-  successfulProponent?: true;
-
+  successfulProponent?: boolean;
```

Comment:
My apologies -- I think I used boolean as a stub value. This should be:

`successfulProposal?: CWUProposalSlim;`

Same for SWU below.